tests/fast-integration/README.md

```diff
@@ -1,9 +1,8 @@
 # Fast integration tests
 
-## Why?
-We need fast, reliable integration tests to speed up Kyma development. The developer productivity is directly proportional to the number of iterations you can make per day/hour/minute. The iteration includes writing code/test, executing tests, refactoring, and executing test after refactoring, and the length of such iteration is a turnaround time. The goal is to decrease the minimal turnaround time from current 90 minutes to less than 10 minutes (ten times). Fast integration tests will solve the problem partially. Other initiatives that are executed in parallel are equally important: switching to k3s starting to reduce kubernetes provisioning time, and implement parallel installation of Kyma components.
+## Overview
 
-## How to speed up integration tests?
+This project provides fast integration tests for Kyma. The goal is to decrease the minimal turnaround time from current 90 minutes to less than 10 minutes (ten times). Fast integration tests will solve the problem partially. Other initiatives that are executed in parallel are equally important: switching to k3s, starting to reduce Kubernetes provisioning time, and implementing parallel installation of Kyma components.
 
 The current integration testing flow looks like this:
 - Build test image(s), push it, ~ 2min/image
```

Comment:
```suggestion
1. Build a test image (or images) and push it (~ 2min/image).
2. Deploy [Octopus](https://github.com/kyma-incubator/octopus/blob/master/README.md) (~1 min).
3. Deploy a test Pod (test image), (~ 1min/image).
4. In many tests, sleep 20 seconds to wait for a sidecar.
5. Deploy the "test scene" (~1 min/image).
6. Execute the test (5 sec/test).
7. Wait for the test completion and collect results (~1 min).

The plan is to keep only two steps:
```

Reply:
Since I couldn't add comments to the lines below, I added them as one big suggestion 🙂 Also, I'm not certain about point no.3 with double parentheses.